guides/release/components/component-arguments-and-html-attributes.md

```diff
@@ -66,19 +66,9 @@ the _browser_ what to do, it's telling your custom tag what to do.
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
         <p>
-          You might be wondering why Ember doesn't use attribute syntax for the
-          component syntax. In the next section, we'll learn that you can put normal
-          HTML attributes on component tags, and they will end up on the HTML
-          element that the component creates.
-        </p>
-        <p>
-          So when you use a normal attribute name, you're putting a normal HTML
-          attribute on an element. When you use an argument (which starts with
-          <code>@</code>), you're providing data to a component.
-        </p>
-        <p>
-          This also means that arguments can pass any kind of data to the
-          component, even though HTML attributes are always strings
```

Comment:
Although interesting, to the common user the example that immediately follows this helps clarify this in a "oh, that's nice and simple" whereas this starts feeling like too much explanation in an aside